src/lib/components/profile/chip/Chip.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+  import cancelIcon from '$lib/icons/cancel.svg'
+  import { expand, shrink } from './expand'
+
+  export let text: string
+  export let key: string
+  export let animation: boolean = true
+  export let deleteHandler: () => void
+
+  let hovered = false
+
+  let chip: HTMLElement
+</script>
+
+{#if animation}
+  <article
+    id={key}
+    bind:this={chip}
+    class="flex max-h-[40px] items-center gap-3 whitespace-nowrap rounded-full bg-ucab-green px-4 py-2 text-brand-white shadow-xl transition-all hover:bg-green-600 hover:shadow-lg hover:shadow-green-200"
+    on:mouseenter={() => (hovered = true)}
+    on:mouseleave={() => (hovered = false)}
+    use:expand={hovered}
+  >
+    <b class="font-normal">{text}</b>
```

Comment:
```suggestion
    <strong class="font-normal">{text}</strong>
```
Esto si quieres resaltar que este texto es más importante que el que está alrededor, si nomás querías resaltarlo sin que fuese más importante como  tal, deja el b